data/fields/seamark/rescue_station/category.json

```
@@ -0,0 +1,19 @@
+{
+    "key": "seamark:rescue_station:category",
+    "type": "semiCombo",
+    "label": "Amenities",
+    "strings": {
+        "options": {
+            "lifeboat": "Lifeboat (on land)",
+            "lifeboat_on_mooring": "Lifeboat (at a mooring)",
```

I think we should title-case also the words in parentheses?

```suggestion
            "lifeboat": "Lifeboat (on Land)",
            "lifeboat_on_mooring": "Lifeboat (at a Mooring)",
```